express-middlewares/expressMiddlewares.js

```diff
@@ -1,8 +1,42 @@
+const express = require('express');
 
-function expressMiddlewares(...args) {
-  return args;
-}
+const app = express();
+app.get('/', (req, res) => {
+  res.send('Hello');
+});
 
-export {
-  expressMiddlewares,
-};
+const languages = 'languages';
+app.get(`/${languages}`, (req, res) => {
+  res.set('middleware-header', languages);
```

Comment:
This question was meant to be done using middleware so that the same function call be called before calling each request.